Editor/Window/AnywherePage.cs

```diff
@@ -26,6 +32,12 @@ public AnywherePage(VisualElement container, StateManager stateManager)
                 new RegisterComputeInput(computeInputContainer, stateManager); 
         }
 
+        private void OnLinkClicked(string url)
```

Comment:
I don't see the need for a utility in this case. Can we use Application.OpenURL inline where required?

Reply:
Removed utility